### PR TITLE
Add within API #53

### PR DESCRIPTION
**Pull request by Dajust**
**What**: Add a `within` API

**Why**: Kindly see #53 

**How**: Exported `getQueriesForElement` as `within` from the index.js file

**Checklist**:

* [x] Documentation
* [x] Tests
* [x] Ready to be merged 
* [x] Added myself to contributors table

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -642,6 +643,22 @@ expect(submitButtons).toHaveLength(3) // expect 3 elements
...
 ```
 
+## `within` API
+
+Sometimes, there is no garauntee that the text, placeholder, or label you want to query is unique on the page. So you might want to explicity tell react-render-dom to get an element **only within** a particular section of the page. `within` is a helper function for this case.
+
+Example: To get the text 'hello' only within a section called 'messages', you could do:
+
+```javascript
+import {render, within} from 'react-testing-library'
```
Comment (kentcdodds) [inline README.md]: Could you change this example to work better in the `dom-testing-library` docs?

```js
import {within} from 'dom-testing-library'

const {getByText} = within(document.body.getElementById('messages'))
const helloMessage = getByText('hello')
```

Also, could you add a note that says:


```md
> Fun fact, this is actually just an alias to the `getQueriesForElement` function!
```

File: src/index.js [hunk 2]
```diff
@@ -16,4 +16,7 @@ export * from './pretty-dom'
 
 // The original name of bindElementToQueries was weird
 // The new name is better. Remove this in the next major version bump.
```
Comment (kentcdodds) [inline src/index.js]: Could you move these two comments to right above the `bindElementsToQueries` export? That's where the comment applies. We'll keep the `within` alias indefinitely.

File: .all-contributorsrc [hunk 3]
```diff
@@ -238,6 +238,17 @@
       "contributions": [
         "example"
       ]
+    },
+    {
+      "login": "Dajust",
+      "name": "Justice Mba ",
```
Comment (sompylasar) [inline .all-contributorsrc]: There's an extra space after your name, typo?

Comment (Dajust) [reply .all-contributorsrc]: Not sure, I think that file is auto-generated.

Comment (kentcdodds) [reply .all-contributorsrc]: You can fix it manually.

You may want to fix it on GitHub too though :wink:

screen shot 2018-06-12 at 11 59 51 am

**Pull request merged**